addons/tmf_loadouts/loadouts/pla_army_2010.hpp

```diff
@@ -0,0 +1,508 @@
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"CUP_LRTV",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"arifle_CTARS_blk_F"};
+	magazines[] = {
+		LIST_6("100Rnd_580x42_Mag_F"),
+		LIST_2("100Rnd_580x42_Mag_Tracer_F"),
+		"CUP_HandGrenade_RGD5",
+		"SmokeShell"
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpackItems[] = {
+		LIST_4("100Rnd_580x42_Mag_F"),
+		"ACE_EntrenchingTool"
+	};
+	linkedItems[] += {"Binocular"};
+};
+
+class rat : car {
+	displayName = "Rifleman (AT)";
+	secondaryWeapon[] = {"CUP_launch_RPG18"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	primaryWeapon[] = {"srifle_DMR_07_blk_F"};
+	scope[] = {"CUP_optic_HensoldtZO_low_3D"};
+	magazines[] = {
+		LIST_11("20Rnd_650x39_Cased_Mag_F"),
+		LIST_2("CUP_HandGrenade_RGD5"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	primaryWeapon[] = {"CUP_lmg_Pecheneg"};
+	magazines[] = {
+		LIST_3("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Green_M"),
+		"CUP_HandGrenade_RGD5",
+		"SmokeShell"
+	};
+};
+
+class mmgac : r {
+	displayName = "MMG Ammo Carrier";
+	backpack[] = {"MNP_B_Carryall_PLA_Basic"};
+	backpackItems[] = {
+		LIST_4("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Green_M")
+	};
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		LIST_2("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Green_M"),
+		"ACE_EntrenchingTool"
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_Kord_Gun_Bag"};
+};
+
+class hmgac : r {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"CUP_B_Kord_Gun_Bag"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CUP_B_Kord_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	secondaryWeapon[] = {"CUP_launch_RPG7V"};
+	secondaryAttachments[] = {"cup_optic_pgo7v2"};
+	magazines[] += {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class matac : r {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"MNP_B_Carryall_PLA_Basic"};
+	backpackItems[] = {
+		LIST_3("CUP_PG7VL_M"),
+		LIST_2("CUP_OG7_M")
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
```

Comment:
Metis' ammo is stored in the tripod, would recommend swapping the gun out for the tripod.